frontend/src/tabs/ExpandableTab.js

```diff
@@ -52,7 +52,11 @@ const ExpandableTab = (props) => {
         {props.title}
       </h1>
       <div className='ExpandableTab__content'>
-        <MainSection {...rest} />
+        <div
+          className={`${props.expanded ? 'NFTs__main_hidden_mobile' : ''} NFTs__main`}
```

Comment:
NFT panel-specific styles shouldn't be handled at this level but in the NFTs tab.